src/stack.rs

```diff
@@ -2022,6 +1706,7 @@ pub unsafe extern "C" fn ts_stack_print_dot_graph(
         }
     }
     fprintf(f, b"}\n\x00" as *const u8 as *const libc::c_char);
+    libc::fclose(f);
```

Comment:
Here the problem: you have to close `f` only if you opened it with `fdopen`

Reply:
You are absolutely right!

Reply:
Fixed, now the tests pass correctly on mac as well